### PR TITLE
Moves the `is_frozen` var to a secondary flag

### PR DESCRIPTION
**Pull request by coiax**
One less var on ALL OBJECTS.

### CODE REVIEW

File: code/game/objects/objs.dm [hunk 1]
```diff
@@ -47,8 +46,8 @@
 
 /obj/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback)
 	..()
-	if(is_frozen)
-		visible_message("<span class = 'danger'><b>[src] shatters into a million pieces!</b></span>")
+	if(HAS_SECONDARY_FLAG(src, FROZEN))
+		visible_message("<span class='danger'>[src] shatters into a million pieces!</span>")
```
Comment (coiax) [inline code/game/objects/objs.dm]: I removed the <b> tags here, because spans > tags

Comment (ChangelingRain) [inline code/game/objects/objs.dm]: This stops being bold then though?

Comment (coiax) [inline code/game/objects/objs.dm]: Yeah, but I don't know why it gets to be more bold than a normal danger span

**Comment by optimumtact** [comment]
This was probably a var due to being in some hot loops, have you profiled the impact HAS secondary flag has on the throwing subsystem?

**Comment by coiax** [comment]
@optimumtact what is a standard test I can do to profile this

**Comment by coiax** [comment]
@optimumtact if it's too expensive, we can demote another flag var and move is_frozen into the main bitflags.

**Comment by optimumtact** [comment]
I dunno smh just throw a bunch of objects around by spwaning a singulo or something and compare it versus master with the profiler

**Comment by coiax** [comment]
What is the profiler? How do you use it?

**Comment by Jordie0608** [comment]
Right click ss13 symbol at top of game window, go to server > profile. Once open it starts recording how long is spent on procs, hit "Refresh" (i think) to see them.

**Comment by coiax** [comment]
According to the profiler, throw_at used 0.00000 CPU per call with the changes made, so I guess either I'm bad at profiling, or it's okay.

**Comment by coiax** [comment]
HAS_SECONDARY_FLAG, if there are no secondary flags is the same as checking if a var is null, so I think that would be... close to the cost of checking a bool? I know that when it comes to Byond efficiency, I know nothing. But I'd hope that it is like, close to normal behaviour.

**Pull request merged**